packages/apollo-react-relay-duct-tape/src/storeObservation/compiledHooks/useCompiledFragment.ts

```diff
@@ -65,6 +65,16 @@ export function useCompiledFragment(
   }, [observableQuery]);
 
   const result = observableQuery.getCurrentResult();
+  if (result.partial) {
+    invariant(
+      false,
+      "useFragment(): Missing data expected to be seeded by the execution query document",
+      // TODO: queryInfo is private member and we can't access it, can we access last diff in other way?
+      // JSON.stringify(
+      //   observableQuery.queryInfo.lastDiff?.diff?.missing?.map((e) => e.path),
+      // ),
```

Comment:
I don’t believe so, maybe just cast to any and be really defensive? It should really aid in debugging

Reply:
Hmmm fair, do you know how we could trigger such scenario in tests just to make sure it works properly?

Reply:
Probably disabling the fragment spreading fix for id would trigger it in people app.

Reply:
@alloy Ok, I can check try that tomorrow. However, I think it's a minor thing so if you agree let's merge the PR. If not I will try to figure out sth tomorrow

Reply:
I didn't manage to trigger it on refetchable PR in TMP, maybe something changed about query/variables we have there. Hope we are ok merging like that and please reopen if you disagree